MSAL/src/native_auth/network/responses/reset_password/MSALNativeAuthResetPasswordChallengeResponse.swift

```diff
@@ -30,7 +30,7 @@ struct MSALNativeAuthResetPasswordChallengeResponse: Decodable, MSALNativeAuthRe
     let challengeType: MSALNativeAuthInternalChallengeType
     let bindingMethod: String?
     let challengeTargetLabel: String?
-    let challengeChannel: MSALNativeAuthInternalChannelType?
```

Comment:
is this field exposed to customers, or is it internal only? In case of the former, this is a breaking change.

Reply:
This class isn't exposed to customer